server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/KeywordsConsistencyOnDelegationMailbox.feature

```diff
@@ -0,0 +1,190 @@
+#***************************************************************
```

Comment:
I will not say it everywhere, but every time **the user** is used, your test is implicit.

Reuse work from https://github.com/linagora/james-project/pull/1053 in here to have explicit jerkins spec with the user name.